code-challenges.test.js

```diff
@@ -51,19 +84,43 @@ const randomNouns2 = ["temperature", "database", "chopsticks", "mango", "deducti
 
 
 // b) Create the function that makes the test pass.
+// Make a function named capitalize
+// use map on the array to grab the element at [0] of each word
+// make certain it lowercases each remaining letter of the word
 
+// function capitalize(arr) {
+//   return arr.map(element => {
+//     return element.charAt(0).toUpperCase() + element.substring(1).toLowerCase()
+//   })
+// }
+//
+// console.log(capitalize(randomNouns1))
 
+// left the console.log in this time to show it's functioning code that does what I described, but it still stays red. I think I'll need additional help with testing, but at least I know how to get results.
 
 // --------------------3) Create a function that takes in a string and logs the index of the first vowel.
 
 // a) Create a test with expect statements for each of the variables provided.
 
+describe("vowelCheck", () => {
+  it("returns the number of the index value with the first vowel", () => {
+    expect(vowelCheck()).toEqual("index value of first vowel")
+  })
+})
+// Got Red! Always Red
+
 const vowelTester1 = "learn"
 // Expected output: 1
 const vowelTester2 = "academy"
 // Expected output: 0
 const vowelTester3 = "challenges"
 // Expected output: 2
 
-
 // b) Create the function that makes the test pass.
+for (var i = 0; i < string.length(); i++) {
+  if (vowelCheck(charAt(i))
+    return i
+  }
+}
+
+// I kept trying to run/figure out this code with no avail. I tried a bunch of different functions to try getting results but nothing I could think of woulf fire correctly. Need second opinion.
```

Comment:
Here's some pseudocode. See if you can write a function based on this
// Create a function with a parameter called string
// because you have a string, iterating using charAt is a good approach with conditional statements
// Hint: conditionals are looking for a true statement. therefore a strict equality should be used to evaluate each value with a vowel
// return index when the conditional is true